Changelog.md

```diff
@@ -1,8 +1,19 @@
+## 1.0.2
+
+- Change implementation to the `prettyprinter` package,
+  using `prettyprinter-compat-ansi-wl-pprint` compatibility package.
+
+  Migrate to `prettyprinter`.
+
+  Note: Going through String won't preserve formatting anymore,
+  as `prettyprinter`s `Show Doc` instance is annotation invariant.
```

Comment:
Is there a way to migrate code affected by this change in behavior?

Reply:
In fact yes. I'll add a note. TL;DR https://hackage.haskell.org/package/prettyprinter-ansi-terminal-1.1.3/docs/Prettyprinter-Render-Terminal.html#v:renderLazy

Reply:
Added